java/src/jmri/jmrit/roster/swing/CopyRosterGroupAction.java

```diff
@@ -86,18 +86,19 @@ public void actionPerformed(ActionEvent event) {
         }
 
         String entry = (String) JOptionPane.showInputDialog(_who,
-                "<html><b>Duplicate roster group</b><br>Enter the name for the new roster group.</html>",
-                "Duplicate Roster Group " + group,
+                Bundle.getMessage("DuplicateRosterGroupNewName"),
+                Bundle.getMessage("DuplicateRosterGroupTitle",group),
                 JOptionPane.INFORMATION_MESSAGE,
                 null,
                 null,
                 null);
-        if (entry == null || entry.equals(Roster.ALLENTRIES)) {
+        entry = entry.trim(); // remove white space around name, also prevent "Space" as a Group name
```

Comment:
The trimming is only called when a new Roster Group is created or an existing one is renamed. I have tested this on some samples. It preserves a name, until you rename a group (use Cancel to escape).
What I hope to prevent is users being confused by one or more (2 spaces etc.) invisible roster group names in the group list. These can be selected from a pop up, but are useless and I think it is a non-destructive improvement to fix this.